src/components/ble/MusicService.h

```diff
@@ -47,7 +47,7 @@ namespace Pinetime {
 
       std::string getAlbum() const;
 
-      int getProgress() const;
+      int getProgress();
```

Comment:
Any reason why this method is not `const` anymore?

Reply:
Nope, it was a leftover change :) thanks.